### PR TITLE
Fixing round_trip_load deprecation warnings (#166)

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -61,6 +61,8 @@ norecursedirs =
     *.egg-info
     .git
 testpaths = tests
+filterwarnings =
```
Comment (eudesbarbosa) [inline setup.cfg:64]: We have a good number of `PendingDeprecationWarning` and `DeprecationWarning` in the test code, we need to clean them before making this change.

**Review by eudesbarbosa — COMMENTED** [review]
Looks good, only issue with `filterwarnings`.

File: setup.cfg (hunk 1 above)
Comment (holtgrewe) [reply setup.cfg:64]: All warnings should be fixed with my patch.

Comment (holtgrewe) [reply setup.cfg:64]: OK, fixed for 3.10, let's see about the CI.

**Review by eudesbarbosa — APPROVED** [review]
All good, thanks for the fixes.

**Pull request merged**